src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetUpdateMethod.cs

```diff
@@ -421,6 +421,17 @@ public override void ExecuteCmdlet()
             HelpMessage = "Used to make a request conditional for the GET and HEAD methods. The server will only return the requested resources if none of the listed ETag values match the current entity. Used to make a request conditional for the GET and HEAD methods. The server will only return the requested resources if none of the listed ETag values match the current entity. Set to '*' to allow a new record set to be created, but to prevent updating an existing record set. Other values will result in error from server as they are not supported.")]
         public string IfNoneMatch { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Enable or disable resilient VM creation.")]
+        public SwitchParameter EnableResilientVMCreate { get; set; }
```

Comment:
should this be a bool as well?

Reply:
ARM review suggested keeping this Switch, because it only supports true and false, and by default should be false.
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1437#issuecomment-2174937582

Reply:
Addressed with design author and changed, Switch is for New-Azvmssconfig and Update is for Update-Azvmss